src/sql/workbench/contrib/executionPlan/browser/widgets/nodeSearchWidget.ts

```diff
@@ -17,10 +17,10 @@ import { SelectBox } from 'sql/base/browser/ui/selectBox/selectBox';
 import { InputBox } from 'sql/base/browser/ui/inputBox/inputBox';
 import { isString } from 'vs/base/common/types';
 
-const CONTAINS_DISPLAY_STRING = localize("queryPlanSearchTypeContains", 'Contains');
-const EQUALS_DISPLAY_STRING = localize("queryPlanSearchTypeEquals", 'Equals');
+const CONTAINS_DISPLAY_STRING = localize("executionPlanSearchTypeContains", 'Contains');
```

Comment:
If you're updating just the IDs you should go through and update the xlf/lcl files as well so the loc team doesn't have to do a bunch of extra work just to update the IDs